jest.config.json

```diff
@@ -1,6 +1,5 @@
 {
-  "preset": "react-native",
```

Comment:
Why are we removing the react-native preset

Reply:
I opted to use `jest-expo` preset because tests were failing due to babel not transforming dependencies in the `node_modules`. To fix it we can add every dependency that is going to be needed while running test in the `transformIgnorePatterns` property but i found that `jest-expo` extends the `react-native` preset and adds the required dependencies for expo too. It will prevent us running into tests errors in the future if we add new dependencies.